blog-api-y-nguyen/Controllers/AuthorsController.cs

```diff
@@ -71,38 +75,38 @@ public IActionResult PutAuthor(int id, Author author)
                     throw;
                 }
             }
-            return NoContent();
+            return Ok();
         }
 
         // POST: api/Authors
         [HttpPost]
-        public ActionResult<Author> PostBlog(Author author)
+        public ActionResult<Author> PostAuthor(Author author)
         {
-            if (_authorRepository.CheckAuthorsIsNull())
+            if (_authorService.CheckAuthorsExist() == false)
```

Comment:
same here, no need to do this check operation :D, also the error should be changed. API users could not understand what meaning it is lol. Should just something like `authors does not exist` is enough